webapp/utils/web_client.jsx

```diff
@@ -95,6 +95,24 @@ class WebClientClass extends Client {
             return success(res.body);
         });
     }
+
+    uploadCertificateFile(file, success, error) {
```

Comment:
Dunno there just to upload the files with system console, do we want this ability in the client.jsx? if that the case happy to move it